jib-gradle-plugin/src/integration-test/java/com/google/cloud/tools/jib/gradle/JibPluginIntegrationTest.java

```diff
@@ -157,6 +157,30 @@ public void testDockerDaemon_defaultTarget() throws IOException, InterruptedExce
             defaultTargetTestProject, "default-target-name:default-target-version"));
   }
 
+  @Test
+  public void testBuildTar_simple() throws IOException, InterruptedException {
+    String imageReference = "gcr.io/jib-integration-testing/simpleimage:gradle";
+    String outputPath =
+        simpleTestProject.getProjectRoot().resolve("build").resolve("jib.tar").toString();
+    BuildResult buildResult = simpleTestProject.build("clean", JibPlugin.BUILD_TAR_TASK_NAME);
+
+    BuildTask classesTask = buildResult.task(":classes");
+    BuildTask jibBuildTarTask = buildResult.task(":" + JibPlugin.BUILD_TAR_TASK_NAME);
+
+    Assert.assertNotNull(classesTask);
+    Assert.assertEquals(TaskOutcome.SUCCESS, classesTask.getOutcome());
+    Assert.assertNotNull(jibBuildTarTask);
+    Assert.assertEquals(TaskOutcome.SUCCESS, jibBuildTarTask.getOutcome());
+    Assert.assertThat(
+        buildResult.getOutput(), CoreMatchers.containsString("Built image tarball at "));
+    Assert.assertThat(buildResult.getOutput(), CoreMatchers.containsString(outputPath));
+
+    new Command("docker", "load", "--input", outputPath).run();
+    Assert.assertEquals(
+        "Hello, world. An argument.\nfoo\ncat\n",
+        new Command("docker", "run", imageReference).run());
```

Comment:
I believe it is the correct image, assuming docker overwrites the previous image with the same tag, since the image gets tagged with whatever `to.image` was set to when it gets `docker load`ed.

Reply:
Can use this for tracking the problem: #274

Reply:
Ah right!

Reply:
should we just tag it with something unique? timestamp millis or image digest/checksum?

Reply:
or should our test fail if a docker load fails. Can we detect that line 178 did not exit `0`?

Reply:
Looks like the `Command` class throws a `RuntimeException` if it doesn't exit 0, so we should be covered there.